sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/ResolveIntegrationTests.cs

```diff
@@ -9,13 +9,19 @@
 
 namespace Azure.Iot.ModelsRepository.Tests
 {
-    public class ResolveIntegrationTests
+    public class ResolveIntegrationTests : ModelRepoRecordedTestBase
     {
-        [TestCase("dtmi:com:example:thermostat;1", TestHelpers.ClientType.Local)]
-        [TestCase("dtmi:com:example:thermostat;1", TestHelpers.ClientType.Remote)]
-        public void ResolveWithWrongCasingThrowsException(string dtmi, TestHelpers.ClientType clientType)
+        public ResolveIntegrationTests(bool isAsync) : base(isAsync)
         {
-            ResolverClient client = TestHelpers.GetTestClient(clientType);
+        }
+
+        [TestCase(ModelRepoTestBase.ClientType.Local)]
+        [TestCase(ModelRepoTestBase.ClientType.Remote)]
+        public void ResolveWithWrongCasingThrowsException(ModelRepoTestBase.ClientType clientType)
+        {
+            string dtmi = "dtmi:com:example:thermostat;1";
```

Comment:
const

Reply:
sure, I will make that change for all of the instances.